File: MVP_Article/MVP_Article_Frontend/ClientApp/src/app/model/empfaenger.ts
import { EntityBase } from "./EntityBase";

export class Empfaenger extends EntityBase{
  public Name: string = "";
  public Vorname: string = "";
  public Strasse: string = "";
  public Hausnummer: string = "";
  public Postleitzahl: number = 0;
  public Ort: string ="";
}

File: MVP_Article/MVP_Article_Frontend/Model/Empfaenger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MVP_Article_Frontend.Model
{
    public class Empfaenger : EntityBase
    {
        public string Name { get; set; }
        public string Vorname { get; set; }
        public string Strasse { get; set; }
        public string Hausnummer { get; set; }
        public int Postleitzahl { get; set; }
        public string Ort { get; set; }
    }
}
